Ten Elephants/Ten Elephants/Models/Meals.swift

```diff
@@ -17,18 +16,16 @@ struct Meals: Decodable {
 struct Meal{
     // always present
     let id: String                 // idMeal
-    
```

Comment:
IMO, it is best to avoid changes in files that are not related to your commit, even if you refactor them. Refactoring of such files can be done as a separate task, if needed